opencti-platform/opencti-front/src/private/components/data/csvMapper/representations/attributes/CsvMapperRepresentationAttributeRefForm.tsx

```diff
@@ -107,7 +93,34 @@ CsvMapperRepresentationAttributeRefFormProps
         return r.id !== fromId;
       }
       return true;
-    });
+    }));
+
+  let options;
+  
+// For both entity types, whether they are related-to or revoked-by,
+// we don't need to resolve the different types, as they can link any entity between them.
```

Comment:
```suggestion
  // For both entity types, whether they are related-to or revoked-by,
  // we don't need to resolve the different types, as they can link any entity between them.
```
To fix eslint errors